docs/source/handle_markdown.py

```
@@ -209,6 +208,24 @@ def process_file(
+    special_links = get_special_links(content)
+    for item in special_links:
+        if not item[0].strip():
+            warnings.warn(
+                f"Empty link in `{file}`. Link name `{item[1]}` link path "
+                f"`{item[0]}`. Will ignore link.",
+                stacklevel=3,
+            )
+            continue
+        if item[0].startswith("http"):  # skip http links and anchors
+            continue
+        else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `continue` statement

Remove unnecessary `else`

(RET507)